src/app/content/studyGuides/reducer.ts

```diff
@@ -8,17 +8,24 @@ import { State } from './types';
 
 export const initialState: State = {
```

Comment:
this whole state is for study guides, the `summary` is supposed to be for the summary data, you can put `open` at the top level next to `isEnabled`